mikupad.html

```diff
@@ -1761,6 +1787,16 @@
 				<${Checkbox} label="Ignore <eos>"
 					disabled=${!!cancel} value=${ignoreEos} onValueChange=${setIgnoreEos}/>
 			</${CollapsibleGroup}>
+			<${CollapsibleGroup} label="Memory">
+				<textarea
+				ref=${memoryArea}
+				readOnly=${!!cancel}
+				placeholder="Anything written here will be injected at the head of the prompt. Tokens here DO count towards the Context Limit."
+				defaultValue=${memoryTokens}
+				value=${memoryTokens}
+				onInput=${onMemoryInput}				

```

Comment:
I don't think this `onMemoryInput` function is necessary, probably you could instead use `(e) => setMemoryTokens(e.target.value)`